src/libs/highlevel/README.md

```diff
@@ -142,7 +142,31 @@ Errors which do not originate inside the high-level API itself are wrapped into
 `ELEKTRA_ERROR_CODE_LOW_LEVEL`. The high-level Error API provides methods (`elektraKDBError*`) to access the properties of the low-level
 error. You can also access the key to which the error was originally attached, as well as any possible low-level warnings.
 
-TODO: explanation of low-level code or removal of this feature for this release?
+To get the original low-level error code, description, severity, group, module and reason you can use these functions:
+```c
+int elektraKDBErrorCode (const ElektraError * error);
```

Comment:
@Piankero Do you want to change anything of these functions in your new error concept?

Reply:
Please give me some time to work out a concept and gather your feedback. I'll try to finish the concept until next weekend at the latest.